File: CS/WinWebSolution.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;
using WinSolution.Module;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(DevExpress.ExpressApp.IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            Department dept = ObjectSpace.CreateObject<Department>();
            dept.Title = "Demo Department";
            LocalEmployee le = ObjectSpace.CreateObject<LocalEmployee>();
            le.Name = "LocalEmployee 1";
            ForeignEmployee fe = ObjectSpace.CreateObject<ForeignEmployee>();
            fe.Name = "ForeignEmployee 1";
            dept.Employees.AddRange(new EmployeeBase[] { le, fe });
            dept.Save();
        }
    }
}

File: VB/WinWebSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl
Imports WinSolution.Module

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As DevExpress.ExpressApp.IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim dept As Department = ObjectSpace.CreateObject(Of Department)()
			dept.Title = "Demo Department"
			Dim le As LocalEmployee = ObjectSpace.CreateObject(Of LocalEmployee)()
			le.Name = "LocalEmployee 1"
			Dim fe As ForeignEmployee = ObjectSpace.CreateObject(Of ForeignEmployee)()
			fe.Name = "ForeignEmployee 1"
			dept.Employees.AddRange(New EmployeeBase() { le, fe })
			dept.Save()
		End Sub
	End Class
End Namespace
